src/models/rfcProps.js

```diff
@@ -182,7 +184,7 @@ const properties = {
 		default: true,
 		defaultValue: {
 			value: '',
-			type: ['HOME', 'VOICE']
+			type: [defaultProfileState]
```

Comment:
Not the expert, but maybe:
```suggestion
			type: [defaultProfileState, 'VOICE']
```

Reply:
right, my bad :)

Reply:
This is tricky, but yes!
Though only `WORK,VOICE` and `HOME,VOICE` are really commonly used here. :)
Should be good for now!